java-generator/core/src/main/java/io/fabric8/java/generator/nodes/AbstractJSONSchema2Pojo.java

```diff
@@ -66,6 +68,26 @@ public static String sanitizeString(String str) {
         return sanitized;
     }
 
+    private static Set<String> classNames = new TreeSet<>();
```

Comment:
I'm not sure I like the approach of using a static variable to keep track of the generated class names.

I think it might be better initialize the variable in the CRGeneratorRunner and then pass it in. (consider the option of running this in parallel for multiple CRDs)